clients/ts/FunctionalTests/package.json

```diff
@@ -32,7 +32,7 @@
     "karma-summary-reporter": "^1.5.0",
     "ts-node": "^4.1.0",
     "typescript": "^3.0.1",
-    "websocket": " ^1.0.26"
+    "ws": " ^6.0.0"
```

Comment:
So I need to get approval for this one?

Reply:
Yes, we are swapping in a pure-JavaScript websocket library instead of the previous WebSocket library we were using which had a native component.

Reply:
Approved.